src/main/lang/Nix.bnf

```diff
@@ -191,7 +191,9 @@ expr_uri ::= URI
 ;{ extends("expr_lookup_path|expr_std_path")=expr_path }
 expr_path ::= expr_lookup_path | expr_std_path
 expr_lookup_path ::= SPATH
-expr_std_path ::= PATH_SEGMENT (PATH_SEGMENT | antiquotation)* PATH_END
+expr_std_path ::= PATH_SEGMENT (PATH_SEGMENT | antiquotation)* PATH_END {
+  mixin="org.nixos.idea.psi.impl.NixExprStdPathMixin"
+}
```

Comment:
question: Do you really need both, a reference contributor and the mixin? As far as I know, this are two different ways to implement references, you shouldn't need both.